examples/blue_pill_f103/itm/main.cpp

```diff
@@ -22,7 +22,7 @@ modm::IOStream stream(itm_device);
 /*
  * Blinks the green user LED with 1 Hz and outputs the loop counter over ITM.
  * SWO pin is PB3. openocd > 0.11.0 is required. Receive data with:
- * scons log-itm fcpu=72000000
+ * scons itm fcpu=72000000
```

Comment:
Is it really necessary to (again) rename the scons commands?

Reply:
No, and I just remembered that I named these commands `log-*` on purpose to make clear that they are somewhat one directional (although the `log-rtt` allows you to send back).